docs/concepts/included_models.rst

```diff
@@ -0,0 +1,286 @@
+including molecules, chemical systems, and alchemical transformations.
+This provides a shared language that tools in the ecosystem use.
+
+You will learn below how the various pieces of **gufe** fit together.
+Generally speaking, :ref:`ChemicalSystems <chemicalsystem>` can be thought of as the *what* or the *nouns* that we are simulating,
+:ref:`Transformations <transformation>` are the *how* or the *verbs* that encode how we are simulating these objects and moving between them,
+and an :ref:`alchemicalnetwork` groups all of these together.
+
+.. image:: ../_static/alchemical_network_diagram.svg
+    :alt: The ``GufeTokenizable`` representation of an ``AlchemicalNetwork``.
+
+
+.. note::
+
+    Some of these classes are designed to be subclassed, and constitute the *extensible points* of the library.
+    These include the following; see the **How-To Guide** for more information on how to extend from each:
+
+    1. :ref:`Component <component>` : :ref:`howto-component`
+    2. :ref:`Protocol <protocol>` : :ref:`howto-protocol`
+    3. :ref:`ComponentMapping <componentmapping>` : :ref:`howto-componentmapping`
+    4. :ref:`AtomMapping <atommapping>` : :ref:`howto-atommapping`
+    5. :ref:`AtomMapper <atommapper>` : :ref:`howto-atommapper`
+
+
+.. _component:
+
+``Component``
+-------------
+
+The :class:`.Component` class represents a portion of a system of molecules,
+with a single ``Component`` capable of representing anything from an individual drug-like molecule, an entire protein, or (even the concept of) a solvent with ions.
+
+These are often used to define the *components* of a :ref:`chemicalsystem`, which form the nodes of an :ref:`alchemicalnetwork`.
+The same ``Component`` may be present within multiple ``ChemicalSystem``\s, such as a :class:`.ProteinComponent` in an ``AlchemicalNetwork`` featuring relative binding transformations between ligands.
+
+As another distinct example: the :class:`.SmallMoleculeComponent` class is used to form the nodes of a :ref:`ligandnetwork`.
+This is useful for representing relative transformations between a series of small molecules without invoking the additional complexity of an :ref:`alchemicalnetwork`.
+
+.. note::
+    The :class:`.Component` is an *extensible point* of the library,
+    and is intended to be subclassed to enable new applications.
+    For details on how to create your own :class:`.Component` classes, see :ref:`howto-component`.
+
+
+.. _chemicalsystem:
+
+``ChemicalSystem``
+------------------
+
+A :class:`.ChemicalSystem` represents a complete system of molecules,
+and is often composed of multiple :ref:`Components <component>`.
+
+These are most often used as nodes of an :ref:`alchemicalnetwork`, with pairs of ``ChemicalSystem``\s connected by :ref:`Transformations <transformation>`.
+Because a ``ChemicalSystem`` functions as a kind of container of :ref:`Components <component>`, more than one ``ChemicalSystem`` can feature the same ``Component``\s.
```

Comment:
```suggestion
Because a ``ChemicalSystem`` functions as a kind of container of references to :ref:`Components <component>`, more than one ``ChemicalSystem`` can feature the same ``Component``\s.
```